climpred/tests/test_hindcastEnsemble_class.py

```diff
@@ -164,6 +164,20 @@ def test_inplace(
     assert hindcast != summed
 
 
+def test_mean_reduce_bias(hindcast_hist_obs_1d):
```

Comment:
Test also with dask?

Reply:
Not needed. We don’t test everything with dask, don’t see a reason why should work with dask

Reply:
I would say this is important for `dask` since it will be applied to the whole prediction ensemble. But if we have `dask` tested for all the metrics and stuff there's no reason this should break. So long as you don't use `.values` excessively here.

Reply:
.values on coords with are 1D isnt a problem anyways